docs/manifest.json

```diff
@@ -394,6 +395,12 @@
     { "title": "Next.js", "icon": "nextjs", "root": "references/nextjs" },
     [
       ["Overview", "/references/nextjs/overview"],
+      "# Guides",
+      ["Read session and user data", "/references/nextjs/read-sessions-data"],
+      ["Add custom sign up and sign in pages", "/references/nextjs/custom-signup-signin-pages"],
+      ["Integrate Clerk into your app tRPC", "/references/nextjs/trpc"],
+      "# References",
```

Comment:
We can make "App Router" and "Pages Router" drop downs/sub-headings under "References".

Reply:
I'm not sure we want to "hide" this content... It looks like the headings are navigable by people. Subheadings would make it clearer that these are subcategories of the references.